docs/JpManualDeploymentGuideScalarDLAuditorOnAWS.md

```diff
@@ -0,0 +1,209 @@
+# AWS での Scalar DL Auditor 導入方法
+
+Scalar DL Auditor は、Ledger の状態を管理し、Scalar DL クライアントがビザンチン障害を検出できるようにするオプションコンポーネントです。  
```

Comment:
```suggestion
Scalar DL Auditor は、ビザンチン故障を検知するためにLedgerと同一の情報を管理するコンポーネントです。
```
Note:
We use fault and failure properly. We follow the following doc for the definition.
http://www.ctestlabs.org/neoacm/1044_2009.pdf

Reply:
I fixed terminology about `Byzantine Fault` in the following commit.

0f9ddae5b1d3375c6f90f2c433aba44d3be0ec23

I will be careful with term `fault` and `failure` when I create the document.